lib/hammer_cli_foreman/architecture.rb

```diff
@@ -11,7 +11,12 @@ class ListCommand < HammerCLIForeman::ListCommand
         field :name, _("Name")
       end
 
-      build_options
+      extend_with(HammerCLIForeman::CommandExtensions::NoTaxonomies.new)
+
+      build_options do |o|
```

Comment:
Can this be somehow moved to the extension?

Reply:
No, I tried it and it does not work, requires some changes in hammer core for it to work.

Reply:
After more detailed review I've come with an idea: extension classes have access to option sources. What if we create a new option source based on `SavedDefaults` for resources that don't support taxonomies and replace `SavedDefaults` source with the new one? This way we will support default behaviour of saved defaults for commands, remove not supported options and will have code defined in one place and reused via extension.